upup/pkg/fi/cloudup/bootstrapchannelbuilder/bootstrapchannelbuilder.go

```diff
@@ -105,6 +105,11 @@ func (b *BootstrapChannelBuilder) Build(c *fi.ModelBuilderContext) error {
 	}
 
 	for _, a := range addons.Spec.Addons {
+		// Older versions of channels that may be running on the upgrading cluster requires Version to be set
+		// We hardcode version to a high version to ensure an update is triggered on first run, and from then on
+		// only a hash change will trigger an addon update.
```

Comment:
Consider mentioning "This can be removed when kops no longer supports k8s 1.22"

Reply:
Unfortunately this follows kops versioning and we don't have any skew/upgrade jump policy. This will have to remain forever unless we change this